src/main/rx/rx.c

```diff
@@ -466,7 +466,7 @@ STATIC_UNIT_TESTED uint16_t applyRxChannelRangeConfiguraton(int sample, const rx
     return sample;
 }
 
-uint8_t getMappedChannel(uint8_t channel)
+static uint8_t getMappedChannel(uint8_t channel)
```

Comment:
(the name is a bit confusing - the function returns rawChannel index - maybe getRawChannel, but that still not perfect - it should contain 'map' too)